mlir/lib/Dialect/Linalg/Transforms/PadTilingInterface.cpp

```diff
@@ -148,6 +186,8 @@ SmallVector<OpFoldResult> linalg::computePaddedShape(
     AffineExpr sumExpr = dims.front();
     for (unsigned i = 1; i < dims.size(); ++i)
       sumExpr = sumExpr + dims[i];
+    // Add 1 to the maximum accessed index and get the final padded size.
+    sumExpr = sumExpr + rewriter.getAffineConstantExpr(1);
```

Comment:
`sumExpr = sumExpr + 1` should just work using this [overload](https://github.com/llvm/llvm-project/blob/28c2c1e06e2dc73df03cfc2d797fa70365d481f2/mlir/include/mlir/IR/AffineExpr.h#L153)

Reply:
```suggestion
    sumExpr = sumExpr + 1;
```